slides.md

```diff
@@ -4,7 +4,138 @@
 
 ---
 # The tools
+
 ---
+# JavaScript basics
+
+To declare a variable, you can either use `const` (for constants), or `let` (for variables that can change over time). Do not use `var`.
```

Comment:
but why not `var`?

Reply:
it's due to the scope in which it becomes accessible. `var` is a variable that can have multiple values as `let`, but it's visible in the outer scope (so: globally if outside a function, all the function inside a function). `let` will be visible only inside the scope it's declared in (e.g., a `for` loop inside a function). Any idea on how could I add this info without being verbose?

Reply:
Something like "Do not use `var` as doing that exposes the variable more than needed/recommended"?

Reply:
how about:

```
TLDR; never use `var`.
Long explanation: blablalblah...
```

?

Reply:
I was about to ask the same as @JozefCeluch . I think having a TLDR and a long version would be great.

Reply:
a52e1fe

Reply:
and 9fabcd5, misused the newlines 😅